apps/src/lib/levelbuilder/CourseOfferingEditor.jsx

```diff
@@ -115,6 +118,50 @@ export default function CourseOfferingEditor(props) {
       : translatedNoneOption;
   };
 
+  const handleVideoSelection = e => {
+    if (e) {
+      const videoUrl = e.value !== 'None' ? e.value : null;
+      setThumbnail(e.thumbnail);
+      updateCourseOffering('video', videoUrl);
+    } else {
+      //Handles clear button
+      setThumbnail(null);
+      updateCourseOffering('video', null);
+    }
+  };
+
+  const noneOption = {
```

Comment:
Nit: for `noneOption` and `renderedOptions`, could you update that these are specific to the `video` field please? Something like `videoNoneOption` and `videoRenderedOptions`